### ISSUE TITLE
Gas handling and multiple fee tokens

### ISSUE THREAD

**Issue by gamarin2** [cosmos/cosmos-economics#29]
How is gas handled in the new sdk? How is gas calculated for each whitelisted fee token? Is is still each validator that sets its own (subjective) exchange rate?

@rigelrozanski @sunnya97

**Comment by rigelrozanski** [comment]
Yeah sunny had mentioned an idea previously which included the ability for validators to kind of set their own exchange rates of various whitelisted tokens, seem to recall there were some gameable problems with this though... Let's restart this discussion
